great_expectations/compatibility/sqlalchemy_and_pandas.py

```diff
@@ -82,48 +82,3 @@ def pandas_read_sql(sql, con, **kwargs) -> pd.DataFrame | Iterator[pd.DataFrame]
             sql = sa.select(sa.text("*")).select_from(sql)
         return_value = pd.read_sql(sql=sql, con=con, **kwargs)
     return return_value
-
-
-def pandas_read_sql_query(
-    sql, con, execution_engine, chunksize=None, **kwargs
-) -> pd.DataFrame:
-    """Suppress deprecation warnings while executing the pandas read_sql_query function.
-
-    Note this only passes params straight to pandas read_sql_query method, please
-    see the pandas documentation
-    (currently https://pandas.pydata.org/docs/reference/api/pandas.read_sql_query.html)
-    for more information on this method.
-
-    If pandas version is below 2.0 and sqlalchemy installed then we suppress
-    the sqlalchemy 2.0 warning and raise our own warning. pandas does not
-    support sqlalchemy 2.0 until version 2.0 (see https://pandas.pydata.org/docs/dev/whatsnew/v2.0.0.html#other-enhancements)
-
-    Args:
-        sql: str or SQLAlchemy Selectable (select or text object)
-        con: SQLAlchemy connectable, str, or sqlite3 connection
-        chunksize: If specified, return an iterator where `chunksize` is the number of rows to include in each chunk.
-        **kwargs: Other keyword arguments, not enumerated here since they differ
-            between pandas versions.
-    Returns:
-        dataframe
-    """
-    if (
-        sa
-        and is_version_greater_or_equal(sa.__version__, "2.0.0")
-        and is_version_less_than(pd.__version__, "2.0.0")
-    ):
-        warn_pandas_less_than_2_0_and_sqlalchemy_greater_than_or_equal_2_0()
```

Comment:
The `pip` resolver probably doesn't allow this to happen anymore, but we probably should keep this warning logic somewhere just in case.

Reply:
I did not know that!